evaluation/log_experiments.py

```diff
@@ -0,0 +1,141 @@
+import os
+
+import comet_ml.integration.pytorch
+from comet_ml import Experiment
+import torch
+import sklearn
+from models.model import BaseModel
+from models.sklearn_model import SklearnModel
+from models.trend_model import TrendModel
+from models.naive_models import AverageYieldModel
+from models.nn_models import BaseNNModel
+
+# get paths
+evaluation_path = os.path.dirname(os.path.realpath(__file__))
+root_path = evaluation_path[:-10]  # remove "evaluation" in the string to get the root path
+
+
+# Configure Comet experiment instance
+_project_name = "AgML-Crop-yield-forecasting"
+
+
+# get api key
+def get_comet_api_key(file=None) -> str:
+    """
+    Function that returns api_key.
+
+    Store the api key under evaluation/api_keys/...
+
+    :param: file: file name that contains one line of the comet api key.
+    """
+
+    api_key = None
+
+    check_api_key_dir = os.path.isdir(os.path.join(evaluation_path, 'api_keys'))
+    file_name = 'comet_ml' if None else file
```

Comment:
Curently always sets file_name to file, even file = None.

Should be:
file_name = 'comet_ml' if file_name == None else file

Reply:
Good catch! Thanks.
I changed it to:

`file_name = 'comet_ml' if not file else file`

since file_name hasn't been defined yet

Reply:
Looks good to me!